motionplan/armplanning/constraint_handler.go

```diff
@@ -134,17 +134,6 @@ func newConstraintHandler(
 		handler.AddStateFSConstraint(name, constraint)
 	}
 
-	switch opt.MotionProfile {
```

Comment:
Could we replace this with an error if `MotionProfile` is set so any script using an arm that's using this to specify a constraint doesn't suddenly silently stop obeying it?

Reply:
No.
We have no users.
Now is the time to make this sane.
We have SO much work to don this code, want to make our lives as sane as possible.
And this is just the beginning.
Going to be removing a ton of options if they aren't used until proven needed.

Reply:
can you check the last commit and make sure i no other place i need to add something.